src/Jackett.Common/Indexers/CardigannIndexer.cs

```diff
@@ -371,16 +371,45 @@ protected string applyGoTemplateText(string template, Dictionary<string, object>
                 if (condition.StartsWith("."))
                 {
                     var conditionResultState = false;
-                    var value = variables[condition];
 
-                    if (value == null)
-                        conditionResultState = false;
-                    else if (value is string)
-                        conditionResultState = !string.IsNullOrWhiteSpace((string)value);
-                    else if (value is ICollection)
-                        conditionResultState = ((ICollection)value).Count > 0;
+                    var equalityRegex = new Regex(@"(.+)+\s+(eq|ne)\s+(.+)");
```

Comment:
Possible typo in the Regex and convert first group to lazy for better performance.
```suggestion
                    var equalityRegex = new Regex(@"(.+?)\s+(eq|ne)\s+(.+)");
```